src/global_logic.js

```diff
@@ -147,19 +147,19 @@ module.exports.makeSummonHeaderString = function (summon, locale) {
 };
 
 module.exports.calcDamage = function (summedAttack, totalSkillCoeff, criticalRatio, enemyDefense, additionalDamage, damageUP, damageLimit) {
-    // ダメージ計算
-    var def = (enemyDefense == undefined) ? 10.0 : enemyDefense;
+    // Damage calculation
+    var def = enemyDefense != undefined ? enemyDefense : 10.0;
     var damage = Math.ceil(Math.ceil(summedAttack / def) * totalSkillCoeff) * criticalRatio;
     var overedDamage = 0;
 
     var limitValues = [[600000, 0.01], [500000, 0.05], [400000, 0.60], [300000, 0.80]];
 
     for (var index = 0; index < 4; index++) {
-        // 減衰ライン算出
+        // Damping line calculation
```

Comment:
as direct translation it's ok. but is that ok as English? it can be "border"
I am a native Japanese, and we use "line" on this context as a short word of "borderline".
Japanglish? (Japanese use English word in a different way)

I found English wiki use "Damage cap" for this term
https://gbf.wiki/Damage_cap